app/Http/Controllers/Three/ReviewsController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Rogue\Http\Controllers\Three;
+
+use Rogue\Models\Post;
+use Illuminate\Http\Request;
+use Rogue\Repositories\Three\PostRepository;
+use Rogue\Http\Controllers\Api\ApiController;
+use Illuminate\Auth\Access\AuthorizationException;
+use Rogue\Http\Transformers\Three\PostTransformer;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+class ReviewsController extends ApiController
+{
+    /**
+     * The post service instance.
+     *
+     * @var Rogue\Repositories\ThreePostRepository
+     */
+    protected $post;
+
+    /**
+     * @var \Rogue\Http\Transformers\Three\PostTransformer
+     */
+    protected $transformer;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract $posts
+     * @return void
+     */
+    public function __construct(PostRepository $post)
+    {
+        $this->post = $post;
+        $this->transformer = new PostTransformer;
+
+        $this->middleware('auth:api');
+    }
+
+    /**
+     * Update a post(s)'s status when reviewed.
+     *
+     * @param Request $request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function reviews(Request $request)
+    {
+        $request->validate([
+            'post_id' => 'required',
+            'status' => 'in:pending,accepted,rejected',
+        ]);
+
+        // Only allow an admin to review the post.
+        if (token()->role() === 'admin') {
+            $review = $request->all();
+            $post = Post::where('id', $review['post_id'])->first();
+            $review['signup_id'] = $post->signup_id;
+            $review['northstar_id'] = $post->northstar_id;
+            $review['old_status'] = $post->status;
+
+            // Append admin's ID to the request for the "reviews" service.
+            $review['admin_northstar_id'] = auth()->id();
+            $reviewedPost = $this->post->reviews($review);
+            $reviewedPostCode = $this->code($reviewedPost);
```

Comment:
I'm not sure these conditionals will ever be used.

Unless it hits an exception, [`$this->post->reviews`](https://github.com/DoSomething/rogue/blob/94a743e4c9a14243e4ca58b8c4d05953016dc594/app/Repositories/Three/PostRepository.php#L154-L181) will always return a post.